src/index.html

```diff
@@ -1,17 +1,174 @@
+      name="registrationForm"
+    >
+
+      <fieldset class="form-section">
+        <legend class="title-form-section">Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input 
+            type="text" 
+            name="surname" 
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input 
+            type="text" 
+            name="name" 
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input 
+            type="number" 
+            name="how-old" 
+            min="1" 
+            max="100" 
+            value="12"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input type="date" name="dateOfBirth" >
+        </label>
+
+        <label class="form-field">
+          I accept the term of the agreement
+          <input type="checkbox" name="acceptTheTerm" >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-section">
+        <legend class="title-form-section">Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input 
+            type="email" 
+            name="mail" 
+            placeholder="email@example.com"
+            multiple
+          >
+        </label>
+
+        <label class="form-field">
+          Password:
+          <input 
+            type="password" 
+            name="password"
+            minlength="10"
+            maxlength="30"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-section">
+        <legend class="title-form-section">An interesting fact about you!</legend>
+
+        <div class="form-field">
+          Do you love cats?            
+          <label>             
+            <input 
+              type="radio" 
+              name="loveCats" 
+              value="Yes"
+            >
+            Yes
+          </label>
+          <label>             
+            <input 
+              type="radio" 
+              name="loveCats" 
+              value="No"
+            >
+            No
+          </label>
+        </div>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input type="color">
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed? 
+          <input type="time" name="time">
+        </label>
+
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select 
+            name="cars" 
+            required 
+            multiple
+          >
+            <option 
+              value=" " 
+              disabled 
+              selected
+            ></option>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
```

Comment:
options should start from the top as was mentioned before. let's move the first option to the bottom
![image](https://user-images.githubusercontent.com/22824431/221567790-30a5695a-64c0-4e0e-b4dd-f7ebb23711b1.png)
```suggestion
            <option value="BMW">BMW</option>
            <option value="Audi">Audi</option>
            <option value="Lada">Lada</option>
            <option 
              value=" " 
              disabled 
              selected
            ></option>
```

Reply:
Thank you ))